packages/at_auth/pubspec.yaml

```diff
@@ -5,19 +5,20 @@ homepage: https://atsign.com/
 repository: https://github.com/atsign-foundation/at_libraries
 
 environment:
-  sdk: '>=3.0.0 <4.0.0'
+  sdk: ^3.6.0
+resolution: workspace
```

Comment:
All packages have `sdk: ^3.6.0` which is the minimum for pub workspaces support.

resolution: workspace means the package's dependencies are resolved centrally.